packages/botonic-react/src/util/dom.js

```diff
@@ -26,9 +29,17 @@ export const scrollToBottom = ({ timeout = 200, behavior = 'smooth' } = {}) => {
   }
 }
 
-export const getWebchatElement = () =>
-  document.getElementById('botonic-webchat')
+export const getWebchatElement = host =>
+  host && host.querySelector('#botonic-webchat')
 
-export const setWebchatElementHeight = newHeight => {
-  getWebchatElement().style.height = newHeight
+// https://stackoverflow.com/questions/9457891/how-to-detect-if-domcontentloaded-was-fired
+export const onDOMLoaded = f => {
+  if (/complete|interactive|loaded/.test(document.readyState)) {
```

Comment:
use `document.readyState.exec(/complete|interactive|loaded/)` instead?

Reply:
it's the same no?